backend/app/services/launch_pad.rb

```diff
@@ -33,19 +33,16 @@ def launch
   def land(consent: true, aborted: nil)
     if aborted
       self.abort(aborted)
-      return { aborted_at: Time.now }
```

Comment:
These changes are mainly to prevent another unnecessary call to the backend from the landing component namely [here](https://github.com/openstax/kinetic/pull/349/files#diff-44e68b70a9d2c00bafd45edf2cb54150beaeee6f98225cdb4c23ba3e7c5fe71fL28)

We can just return the updated launched study which should have all of the appropriate completed/aborted information